app/pipeline/faq_ingestion_pipeline.py

```
@@ -0,0 +1,146 @@
+import logging
+import threading
+from asyncio.log import logger
+from typing import Optional, List, Dict
+from langchain_core.output_parsers import StrOutputParser
+from openai import OpenAI
```

Ruff (0.8.2)

`openai.OpenAI` imported but unused

Remove unused import: `openai.OpenAI`

(F401)